modular/executor/executor.go

```diff
@@ -110,12 +110,12 @@ func (e *ExecuteModular) AskTask(ctx context.Context, limit corercmgr.Limit) {
 		if e.omitError(err) {
 			return
 		}
-		log.CtxWarnw(ctx, "failed to ask task", "remaining", limit.String(), "error", err)
+		log.CtxErrorw(ctx, "failed to ask task", "remaining", limit.String(), "error", err)
 		return
 	}
 	// double confirm the safe task
 	if askTask == nil {
-		log.CtxWarnw(ctx, "failed to ask task, dangling pointer",
+		log.CtxErrorw(ctx, "failed to ask task, dangling pointer",
```

Comment:
fixed as`"failed to ask task due to dangling pointer"`

Reply:
Updated.